repl/appengine/web/src/app/repl-console/repl-console.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="repl-console-wrapper" >
+<h1>REPL Console</h1>
+<p>Add statements to evaluate.</p>
+<p>Do not include sensitive data.</p>
+
+<div class="repl-input-block"
+    (keydown)="onEnter($event)">
+    <div class="statement-block" *ngFor="let response of lastEvaluate.responses; let i = index">
+        <mat-form-field class="statement-container">
+            <mat-label>{{i + 1}}</mat-label>
+            <input matInput class="repl-stmt-input"
+              value="{{lastRequest.commands[i]}}"
+              [attr.data-stmt-index]="i"
+              (keydown.ArrowUp)="handleUp(i)"
+              (keydown.ArrowDown)="handleDown(i)">
+            <mat-hint>
```

Comment:
IIRC, the default font size for material hint is very tiny. Since we're using it as a place for an output, it might make sense to adjust the font's formatting accordingly.